x/liquidstake/keeper/rebalancing.go

```diff
@@ -210,7 +210,7 @@ func (k Keeper) AutocompoundStakingRewards(ctx sdk.Context, whitelistedValsMap t
 
 	// re-staking of the accumulated rewards
 	cachedCtx, writeCache := ctx.CacheContext()
-	delegableAmount := proxyAccBalance.Amount.Sub(autocompoundFee.Amount)
```

Comment:
this is incorrect. the `proxyAccBalance` should have rewards auto claimed from staking/ unstaking/ redelegate transactions. And they are not accounted for in `totalRewardsWithdrawn`.